MCprep_addon/tracking.py

```diff
@@ -386,35 +407,68 @@ def set_tracker_json(self):
 		if self._tracker_json is None:
 			raise ValueError("tracker_json is not defined")
 
+		# Placeholder struc before saving to class
+		_json = {
+			"install_date": None,
+			"install_id": None,
+			"enable_tracking": False,
+			"status": None,
+			"metadata": {}
+		}
+
+		valid_tracker = False
 		if os.path.isfile(self._tracker_json) is True:
 			with open(self._tracker_json) as data_file:
-				self.json = json.load(data_file)
+				jdata = json.load(data_file)
+
+			_json = jdata
+			if self._verbose:
+				print(f"{self._addon}: Read in json settings from tracker file")
+			if jdata.get("install_id") in SKIP_IDS:
+				valid_tracker = False
+				_json["install_id"] = None
+				_json["status"] = "invalid_id"
 				if self._verbose:
-					print(self._addon+": Read in json settings from tracker file")
-		else:
-			# set data structure
-			self.json = {
-				"install_date":None,
-				"install_id":None,
-				"enable_tracking":False,
-				"status":None,
-				"metadata":{}
-			}
+					print(f"{self._addon}: Skip ID detected, treat as new")
```

Comment:
```suggestion
					print(f"{self._addon}: ID in skip list detected, treating as new")
```

Reply:
With the new "Invalid ID" naming, I think this reads better now: `Invalid ID detected, treat as new`